packages/manager/src/features/CloudPulse/Widget/components/CloudPulseAggregateFunction.tsx

```diff
@@ -51,27 +57,34 @@ export const CloudPulseAggregateFunction = React.memo(
       setSelectedAggregateFunction,
     ] = React.useState<AggregateFunction>(defaultValue);
 
+    const theme = useTheme();
+
     return (
-      <StyledWidgetAutocomplete
-        isOptionEqualToValue={(option, value) => {
-          return option.label == value.label;
-        }}
-        onChange={(e, selectedAggregateFunc: AggregateFunction) => {
-          setSelectedAggregateFunction(selectedAggregateFunc);
-          onAggregateFuncChange(selectedAggregateFunc.label);
-        }}
-        textFieldProps={{
-          hideLabel: true,
-        }}
-        autoHighlight
-        disableClearable
-        fullWidth={false}
-        label="Select an Aggregate Function"
-        noMarginTop={true}
-        options={availableAggregateFunc}
-        sx={{ width: '100%' }}
-        value={selectedAggregateFunction}
-      />
+      <CloudPulseTooltip title={'Aggregation function'}>
+        <Autocomplete
+          getOptionLabel={(option) => {
+            return convertStringToCamelCasesWithSpaces(option.label); // options needed to be display in Caps first
+          }}
+          isOptionEqualToValue={(option, value) => {
+            return option.label === value.label;
+          }}
+          onChange={(e, selectedAggregateFunc) => {
+            setSelectedAggregateFunction(selectedAggregateFunc);
+            onAggregateFuncChange(selectedAggregateFunc.label);
+          }}
+          textFieldProps={{
+            hideLabel: true,
+          }}
+          autoHighlight
+          disableClearable
+          fullWidth={false}
+          label="Select an Aggregate Function"
+          noMarginTop={true}
+          options={availableAggregateFunc}
+          sx={getAutocompleteWidgetStyles(theme)}
```

Comment:
You can actually pass a  function to `sx`.  The theme will be passed automatically, so you can even remove the `useTheme` hook above

```suggestion
          sx={getAutocompleteWidgetStyles}
```